vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -76,6 +76,20 @@
     path: "{{ root_folder }}/common/symmetricKeyWithNewLine.decrypted"
     state: absent
 
+- name: remove pip2 that might come up along OS        
+  become: yes
```

Comment:
The name says "remove pip2" but the task removes pip3... the name and the task are not agreeing.